internal/provider/harness.go

```diff
@@ -44,6 +44,11 @@ type RegistryResourceTlsModel struct {
 	CaFile   types.String `tfsdk:"ca_file"`
 }
 
+type ContainerResources struct {
```

Comment:
to stay consistent with other tooling, can we nest this?

```suggestion
type ContainerResources struct {
  Memory ContainerMmemoryResources `tfsdk:"memory"`
}
```

Reply:
Do you want to mimic the Kubernetes structure, like this?
```terraform
resources = {
  memory = {
    requests = "1G"
    limit = "2G"
  }
}
```